pkg/issuer/acme/dns/route53/route53.go

```diff
@@ -192,7 +199,7 @@ func (r *DNSProvider) changeRecord(action, fqdn, value string, ttl int) error {
 	if err != nil {
 		if awserr, ok := err.(awserr.Error); ok {
 			if action == route53.ChangeActionDelete && awserr.Code() == route53.ErrCodeInvalidChangeBatch {
-				klog.V(5).Infof("ignoring InvalidChangeBatch error: %v", err)
+				r.log.V(logf.DebugLevel).Info("ignoring InvalidChangeBatch error:", err)
```

Comment:
Either use `Infof("....%v")` or use a `"err", err` key value pair for structured log.